src/planning_core/src/planning_process/planning_process.cpp

```diff
@@ -10,12 +10,148 @@ namespace Planning {
     // Read config file for planning process
     configReaderProcess = std::make_unique<ConfigReader>();
     configReaderProcess->readProcessConfig();
-    auto obsDis = configReaderProcess->getProcess().obs_dis_;
+    obsDis = configReaderProcess->getProcess().obs_dis_;
+
+    // create PNCMap & Global Path Client
+    pncMapClient = this->create_client<base_msgs::srv::PNCMapService>("pnc_map_server");
+    globalPathClient = this->create_client<base_msgs::srv::GlobalPathService>("global_path_server");
   }
 
   boolean PlanningProcess::process()
   {
+    // set 1 second timer to wait rviz2 & xacro to be ready
+    rclcpp::Rate rate(1s);
+    rate.sleep();
+
+    if (initPlanning() == false)
+    {
+      RCLCPP_ERROR(this->get_logger(), "planning init failed!");
+      return false;
+    }
+
+    // planning main process
+
```

Comment:
Here we  can add some prints to help debug
```suggestion
    else
    {
      RCLCPP_INFO(this->get_logger(), "init complete. Start planning process main function!");
    }
```